index.html

```diff
@@ -90,136 +137,116 @@ <h2 class="multi-post">Multi-Post stories</h2>
+          <li><img src="./Images/Union0.png" alt="downward icon"></li>
+        </ul>
 
-      <ul class="buttons">
-        <li><button class="html-btn">html</button></li>
-        <li><button class="css-btn">css</button></li>
-        <li><button class="js-btn">javascript</button></li>
-      </ul>
-      <div class="project-btn"><button class="pro-btn">See Project</button></div>
+      </div>
     </div>
   </section>
 
   <section>
-    <div class="grid-container2">
-      <h1 class="about-heading">About Myself</h1>
-      <p class="my-intro">Hello I’m a software developer! I can help you build a product , feature or website Look
-        through some of my work and experience! If you like what you see and have a project you need coded, don’t
-        hestiate to contact me.</p>
-      <p class="connect-to">LET'S CONNECT</p>
-      <ul class="icons-2">
-        <li><a href="#"><img src="./Images/github.png" alt="icon"></a></li>
-        <li><a href="https://www.linkedin.com/in/khawanin-ali-zada-93777b21a/"><img src="./Images/linkedin.png"
-              alt="linkedin icon"></a></li>
-        <li><a href="#"><img src="./Images/vector.png" alt="icon"></a></li>
-        <li><a href="https://twitter.com/AliKhawanin?t=WjCWTIUHm2vvE734-wThJQ&s=03"><img src="./Images/twitter.png"
-              alt="twitter icon"></a></li>
-        <li><a href="mailto:khawaninalizada21@gmail.com"><img src="./Images/gmail.png" alt="Mail icon"></a></li>
-      </ul>
-      <div><button class="resume">Get my resume</button></div>
-
-      <ul class="languages">
-        <li class="lang">
-          <h2>Languages</h2>
-        </li>
-        <li><img src="./Images/Union1.png" alt="downward icon"></li>
-      </ul>
-
-      <ul class="web-langs">
-        <li><img src="./Images/Frame9.png" alt="js"></li>
-        <li><img src="./Images/Frame10.png" alt="html"></li>
-        <li><img src="./Images/Frame11.png" alt="css"></li>
-      </ul>
-
-      <ul class="framework">
-        <li>
-          <h2>Frameworks</h2>
-        </li>
-        <li><img src="./Images/Union0.png" alt="downward icon"></li>
-      </ul>
-
-      <ul class="skills">
-        <li>
-          <h2>Skills</h2>
-        </li>
-        <li><img src="./Images/Union0.png" alt="downward icon"></li>
-      </ul>
-
+    <div class="form-background">
+      <h2 class="form-heading">Contact me</h2>
+      <p class="form-content">If you have an application you are interested in developing, a feature that you need built
+        or a project that needs coding. I’d love to help with it.</p>
+      <form class="contact-form" action="https://formspree.io/f/mvolvljr" method="POST">
+        <input class="name" type="text" name="firstname" maxlength="30" placeholder="Please enter your name " required>
+        <input class="email" type="email" name="email" placeholder="Please enter your email" required>
+        <textarea class="message" name="message" maxlength="500" placeholder="Write your message here"
```

Comment:
Please, also modify the contact me section to match the design on figma. Check the comment above for more reason you should stick to your given design.

Your design:
![image](https://user-images.githubusercontent.com/20554583/187010263-a9b809fb-730c-4177-8725-faa01d90b1be.png)


Figma design:
![image](https://user-images.githubusercontent.com/20554583/187010295-54ce6008-e5bd-4ac2-99bd-430e8c2152d2.png)